scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -126,6 +126,15 @@ class PublishVisibility(Enum):
     UNLISTED = 'unlisted'
 
 
+class SystemRegisty:
+    """Prod System registries where assets lie."""

```

Comment:
The use of "lie" here feels odd to me.
```suggestion
    """Prod system registries."""
```